crates/biome_cli/src/execute/traverse.rs

```diff
@@ -32,17 +33,28 @@ pub(crate) fn traverse(
     execution: &Execution,
     session: &mut CliSession,
     cli_options: &CliOptions,
-    inputs: Vec<OsString>,
+    mut inputs: Vec<OsString>,
 ) -> Result<(TraversalSummary, Vec<Error>), CliDiagnostic> {
     init_thread_pool();
-    if inputs.is_empty()
-        && execution.as_stdin_file().is_none()
-        && !cli_options.no_errors_on_unmatched
-    {
-        return Err(CliDiagnostic::missing_argument(
-            "<INPUT>",
-            format!("{}", execution.traversal_mode),
-        ));
+
+    if inputs.is_empty() {
+        match execution.traversal_mode {
+            TraversalMode::Check { .. }
+            | TraversalMode::Lint { .. }
+            | TraversalMode::Format { .. }
+            | TraversalMode::CI { .. } => match current_dir() {
+                Ok(current_dir) => inputs.push(current_dir.into_os_string()),
```

Comment:
Could we use the relative path instead of the absolute path here? It seems that our globs don't work well with absolute paths.

Reply:
If we remove the prefix of the `current_dir`, I think it would be the relative path. I'll check that.

Reply:
This could also improve hyperlinks in the CLI: generally IDE expects relative paths.